elastic/kibana
# [Security Solution][Exceptions] - Fix modal closure on addition of first exception list added to a rule

## Summary

- Fixes 🐛  https://github.com/elastic/kibana/issues/127711
- Re-enables exception cypress tests that were skipped
- Cleans up esarchiver to remove two large folders that were previously used for exceptions tests, moved to use smaller, more focused samples

When an exception is added to a rule, prior to that rule ever having any associated exception lists, a new exception list is created and added to the rule. This was also calling on a rule re-fetch, which if the exception was being added from the alerts table meant that the entire alerts table was being re-rendered --> flyout would close. 

The rule re-fetch is unnecessary at that point. The hooks that maintain the logic for associating the exception list to the rule and adding new exceptions also take version control into account. Instead, moved notification of rule change to occur on successful addition of an exception item.

To test:
- Create a new rule
- Wait for alerts to show
- Select to add an exception from alert action menu
- Note whether that results in modal closing (it shouldn't anymore)
- Add an exception
- Ensure it shows up when you visit the Rule Details --> Exceptions (tab)

### Checklist
